app/api/delivery_api/controllers/line_item_api.rb

```diff
@@ -0,0 +1,28 @@
+module DeliveryApi
+  module Controllers
+    class LineItemApi < Root
+      helpers DeliveryApi::Helpers::CartHelpers
+      helpers do
+        def line_item
+          current_cart.line_items.find(params[:line_item_id])
+        end
+      end
+
+      resources :line_items do
+        before { authenticate! }
+
+        route_param :line_item_id, type: Integer do
+          desc 'Change product quantity.'
+          params { requires :quantity, type: Integer }
+          patch :quantity do
+            item = line_item.change_quantity(declared_params[:quantity])
+            present(message: "Quantity updated to #{params[:quantity]}") if item
```

Comment:
Frontend always requires a machine-readable response which easies to process. 
So, I would suggest something like:
```
command = line_item.change_quantity(declared_params[:quantity])
present(message: "Quantity updated to #{declared_params[:quantity]}", quantity: declared_params[:quantity]) if command
```

Reply:
> Frontend always requires a machine-readable response which easies to process.
> So, I would suggest something like:
> 
> ```
> command = line_item.change_quantity(declared_params[:quantity])
> present(message: "Quantity updated to #{declared_params[:quantity]}", quantity: declared_params[:quantity]) if command
> ```

I like it. Fixed